code/modules/research/xenoarchaeology/artifact/artifact_unknown.dm

```diff
@@ -1,4 +1,5 @@
 var/list/excavated_large_artifacts = list()
+var/list/destroyed_large_artifacts = list()
```

Comment:
I feel like both of these lists should be inside SSxenoarch instead of global scope

Reply:
large artifacts placed by mappers will runtime as SSxenoarch isn't already there when they try to be added to excavated_large_artifacts on New()

Reply:
That subsystem is only really about placing stuff at roundstart

Reply:
adding to excavated_large_artifacts can be done on Initialize(), to do this SSxenoarch needs to initialize before SSobj, that should do it. I think it has its benefits, like for example defining a proper place for some artifact tracking/management. It's also easier to debug for problems on live, unless someone figured out how to access global variables. It's only a suggestion though